src/analysis/ContextBar.test.ts

```diff
@@ -686,32 +677,27 @@ describe('ContextBar - actions', () => {
   it('clicking Terminal will attempt to start currently stopped runtime', async () => {
     // Arrange
     const mockRuntimesStartFn = jest.fn();
-    const mockRuntimeWrapper = jest.fn(() => ({
+    type RuntimesContract = AjaxContract['Runtimes'];
```

Comment:
Looks like this is used a few times... suggest moving it up to the top of the file where AjaxContract is defined.